src/Sulu/Bundle/MediaBundle/Tests/Unit/Media/Manager/MediaManagerTest.php

```diff
@@ -210,6 +221,24 @@ public function testDelete()
         $this->mediaManager->delete(1, true);
     }
 
+    public function testSpecialCharacterFileName()
+    {
+        $fileName = 'aäüßa';
```

Comment:
Do you know the [data providers from PHPUnit](https://phpunit.de/manual/3.7/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.data-providers)? If you use them it would be quite easy to add more tests cases, e.g. in this one it would be interesting to also test file names with dots.